Tasks/SonarQubePreBuild/SonarQubePreBuild.ps1

```diff
@@ -13,51 +13,34 @@ param(
 
 Write-Verbose "Starting SonarQube Pre-Build Setup Step"
 
-Write-Verbose -Verbose "connectedServiceName = $connectedServiceName"
-Write-Verbose -Verbose "projectKey = $projectKey"
```

Comment:
I removed -Verbose as I have received feedback from the TFS Build team. Using -Verbose will cause the text to be always outputted. I am still trying to understand how to configure the build so that it outputs "Debug" messages.